src/style.css

```diff
@@ -1,3 +1,68 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+
+  --activation-color: #00ACDC;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0 40px;
+}
+
+.nav__item {
+  margin: 0;
+  padding: 0 10px;
```

Comment:
The first nav item shouldn't have a left padding and the last one shouldn't have a right padding